source/install-desktops/install-desktop-app.rst

```diff
@@ -0,0 +1,103 @@
+.. _install-desktops-install-desktop-app:
+
+Install Desktop App
+===================
+
+No we will go through installing the provided `bc_desktop`_ app as well as
```

Comment:
1. Fix typo: **Now we will install the...**
2. We don't mention why it is prefixed `bc_`. Maybe a separate section?